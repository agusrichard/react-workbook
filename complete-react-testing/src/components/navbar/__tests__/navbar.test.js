import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import { BrowserRouter as Router } from 'react-router-dom'


import Navbar from '../navbar.components'

it('Snapshot navbar', () => {

  const wrapper = mount(
    <Router>
      <Navbar />
    </Router>
  )

  expect(toJson(wrapper)).toMatchSnapshot()
})