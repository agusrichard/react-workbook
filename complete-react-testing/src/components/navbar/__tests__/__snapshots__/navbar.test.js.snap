// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot navbar 1`] = `
<BrowserRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "length": 1,
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
  >
    <Navbar>
      <styled.nav>
        <nav
          className="sc-gKAaRy kvcYfn"
        >
          <ul>
            <NavList>
              <styled.li
                key="0"
              >
                <li
                  className="sc-iCoGMd cKjKIQ"
                >
                  <Link
                    to="/home"
                  >
                    <LinkAnchor
                      href="/home"
                      navigate={[Function]}
                    >
                      <a
                        href="/home"
                        onClick={[Function]}
                      >
                        Home
                      </a>
                    </LinkAnchor>
                  </Link>
                </li>
              </styled.li>
              <styled.li
                key="1"
              >
                <li
                  className="sc-iCoGMd cKjKIQ"
                >
                  <Link
                    to="/about"
                  >
                    <LinkAnchor
                      href="/about"
                      navigate={[Function]}
                    >
                      <a
                        href="/about"
                        onClick={[Function]}
                      >
                        About
                      </a>
                    </LinkAnchor>
                  </Link>
                </li>
              </styled.li>
              <styled.li
                key="2"
              >
                <li
                  className="sc-iCoGMd cKjKIQ"
                >
                  <Link
                    to="/blog"
                  >
                    <LinkAnchor
                      href="/blog"
                      navigate={[Function]}
                    >
                      <a
                        href="/blog"
                        onClick={[Function]}
                      >
                        Blog
                      </a>
                    </LinkAnchor>
                  </Link>
                </li>
              </styled.li>
              <styled.li
                key="3"
              >
                <li
                  className="sc-iCoGMd cKjKIQ"
                >
                  <Link
                    to="/project"
                  >
                    <LinkAnchor
                      href="/project"
                      navigate={[Function]}
                    >
                      <a
                        href="/project"
                        onClick={[Function]}
                      >
                        Project
                      </a>
                    </LinkAnchor>
                  </Link>
                </li>
              </styled.li>
              <styled.li
                key="4"
              >
                <li
                  className="sc-iCoGMd cKjKIQ"
                >
                  <Link
                    to="/contact"
                  >
                    <LinkAnchor
                      href="/contact"
                      navigate={[Function]}
                    >
                      <a
                        href="/contact"
                        onClick={[Function]}
                      >
                        Contact
                      </a>
                    </LinkAnchor>
                  </Link>
                </li>
              </styled.li>
            </NavList>
          </ul>
        </nav>
      </styled.nav>
    </Navbar>
  </Router>
</BrowserRouter>
`;
